<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRegisterPeer</class>
 <widget class="QDialog" name="DlgRegisterPeer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="2">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_unregister">
       <property name="text">
        <string>Unregister</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_register">
       <property name="text">
        <string>Register</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_password">
       <property name="toolTip">
        <string>Password of Subutai Console</string>
       </property>
       <property name="text">
        <string>Console Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lne_peername"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lne_username"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_peer_name">
       <property name="toolTip">
        <string>Peer name for Bazaar</string>
       </property>
       <property name="text">
        <string>Peer name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lne_password"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_username">
       <property name="toolTip">
        <string>Username for Subutai Console.</string>
       </property>
       <property name="text">
        <string>Console Username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_peer_scope">
       <property name="toolTip">
        <string>Peer scope</string>
       </property>
       <property name="text">
        <string>Peer Scope</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmb_peer_scope">
       <item>
        <property name="text">
         <string>Public</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Private</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shared</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="lbl_info">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
